code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -715,7 +715,7 @@ GLOBAL_DATUM_INIT(sdql2_vv_statobj, /obj/effect/statclick/SDQL2_VV_all, new(null
 					SDQL2_TICK_CHECK
 					SDQL2_HALT_CHECK
 	if(islist(obj_count_finished))
-		obj_count_finished = obj_count_finished.len
+		obj_count_finished = length(obj_count_finished)
```

Comment:
Handling list case is kinda pointless here, it's not iterated or anything, might as well be simple number. I guess might be done in another PR.